src/test/kotlin/no/nav/syfo/client/person/adressebeskyttelse/AdressebeskyttelseClientSpek.kt

```diff
@@ -0,0 +1,66 @@
+package no.nav.syfo.client.person.adressebeskyttelse
+
+import io.ktor.client.*
+import io.ktor.client.engine.mock.*
+import io.ktor.client.features.json.*
+import io.ktor.http.*
+import io.mockk.every
+import io.mockk.justRun
+import io.mockk.mockk
+import io.mockk.verify
+import kotlinx.coroutines.runBlocking
+import no.nav.syfo.application.cache.RedisStore
+import no.nav.syfo.testhelper.UserConstants.ARBEIDSTAKER_FNR
+import org.amshove.kluent.shouldBeEqualTo
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+
+class AdressebeskyttelseClientSpek : Spek({
+
+    val cacheKey = "person-adressebeskyttelse-${ARBEIDSTAKER_FNR.value}"
+    val mock = MockEngine {
+        respond(
+            "{ \"beskyttet\": false }",
+            HttpStatusCode.OK,
+            headersOf("Content-Type", ContentType.Application.Json.toString())
+        )
+    }
+
+    val httpClientMock = HttpClient(mock) {
+        install(JsonFeature) {
+            serializer = JacksonSerializer()
+        }
+    }
```

Comment:
Om du vil mock'e "ekte" server istedet for å mocke client, så kan man bruke `EmbeddedServer.embeddedServer` i kotlin. F.eks ved å gjenbruke `SyfopersonMock` som er satt opp som mockServer for Syfoperson sine endepunket, hvor Adresebeskyttelse er et av dem.